src/index.js

```diff
@@ -3,9 +3,12 @@
 const Discord = require("discord.js");
 const fs = require("fs");
 
+const ora = require("ora");
 const log = require("./logger.js");
-const allowModify = require("./util/allowModify.js");
 const config = require("../config.js")();
+const chalk = require("chalk");
+
+log.info('正在啓動機器人客戶端');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/blackcatbot/blackcat/pullRequest?prid=8779392)
```suggestion
log.info("正在啓動機器人客戶端");
```